feed/history/boost_1_68_0.qbk

```diff
@@ -45,6 +50,9 @@
   * Support for multiple long names for an option, thanks to Eyal Rozenberg
     ([@https://github.com/boostorg/program_options/pull/53 PR#53])
 
+* [phrase library..[@/libs/python/ Python]:]
+  * Bug fixes to correct autolink support (Windows) ([ticket 193])
```

Comment:
I think correct link is GitHub Issue instead of Trac.
https://github.com/boostorg/python/issues/193

Reply:
Indeed, thanks for the review !